sherpa/sim/__init__.py

```diff
@@ -668,6 +668,10 @@ def get_draws(self, fit, sigma, niter=1000, cache=True, rng=None):
            values.
         niter : int, optional
            The number of draws to use. The default is ``1000``.
+        cache : bool, optional
```

Comment:
As an aside: this is somewhere I wanted to look at the cache handling, given 

- #2166 
- #2774

but it looks like the code already "should work", which is nice.

There's the difference between turning on/off the cache during a fit/mcmc run/error analysis and the general purpose "always cache" behavior which we are struggling to document/handle correctly, but that's a different concern.